XenAdmin/Properties/Resources.resx

```diff
@@ -127,9 +127,6 @@
   <data name="sl_add_storage_system_16" type="System.Resources.ResXFileRef, System.Windows.Forms">
     <value>..\Images\sl_add_storage_system_16.png;System.Drawing.Bitmap, System.Drawing, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a</value>
   </data>
-  <data name="homepage_background" type="System.Resources.ResXFileRef, System.Windows.Forms">

```

Comment:
If you're going to remove these images from the resources, you can remove them from the source tree too.